src/parser/mod.rs

```diff
@@ -5861,6 +5863,22 @@ impl<'a> Parser<'a> {
         })
     }
 
+    pub fn parse_drop_extension(&mut self) -> Result<Statement, ParserError> {
```

Comment:
Since the function is being made a public function, could we include a description for it?

Reply:
Added